samples/policy-off.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: install.istio.io/v1alpha2
```

Comment:
A general comment about the sample configuration files name, would it be more clear to rename them as {base-profile}-{customization part}, e.g: sds-policy-off

Reply:
Done.